src/m365/booking/commands/business/business-get.spec.ts

```diff
@@ -41,6 +41,13 @@ describe(commands.BUSINESS_GET, () => {
     sinon.stub(session, 'getId').returns('');
 
     auth.service.connected = true;
+    sinon.stub(Cli.getInstance(), 'getSettingWithDefaultValue').callsFake((settingName: string, defaultValue: any) => {
+      if (settingName === 'prompt') {
+        return false;
+      }
+
+      return defaultValue;
+    });
```

Comment:
If it does, do remove it 😀

Reply:
it does. I need to check in other places then. As I think it is added in multiple places

Reply:
Hi @Adam-it, the default prompt setting should be true, so are you sure this should be removed? Maybe it's working on your end because of interference with an actual prompt setting?

Reply:
I will check how it behaves when I have prompt to false and prompt to true.
Thanks for the tip